docs/src/user-guide/guides-using-object-storage/index.md

```diff
@@ -24,8 +24,7 @@ will be added in a future release.
 
 1. An S3 bucket and [key prefix][aws-key-prefixes] containing the logs you wish to compress.
```

Comment:
Only the first key prefix has a link. This makes sense to me but wanted to confirm this is intentional

Reply:
Yeah, I figure the user will read the first link if they're confused and then they shouldn't need it repeated.